vars/pullGitRepositoryToSapCloudPlatformAbapEnvironment.groovy

```diff
@@ -0,0 +1,185 @@
+import static com.sap.piper.Prerequisites.checkScript
+import com.sap.piper.ConfigurationHelper
+import com.sap.piper.GenerateDocumentation
+import com.sap.piper.JenkinsUtils
+import com.sap.piper.Utils
+import groovy.json.JsonSlurper
+import hudson.AbortException
+import groovy.transform.Field
+import org.jenkinsci.plugins.workflow.steps.FlowInterruptedException
+
+@Field def STEP_NAME = getClass().getName()
```

Comment:
we should use a name which complies with the pattern typically used:

`<tool><verb><noun>`, e.g. `sapABAPPullGitRepo`

Reply:
That‘s pretty long ... and due to the technical nature I would suggest to use abbreviations.

Going forward, we should think about a crisp name which can also be easily used when using the golang binary.

Reply:
new Proposal: abapEnvironmentPullGitRepo
What do you think?

Reply:
That is OK from my side ...
If we want to make it even a bit shorter we could think of `abapEnvPullGitRepo`.

Anyway, your proposal is also fine with me.